content/en/platform/corda/5.0-beta/corda-cli-reference/overview.md

```diff
@@ -7,4 +7,4 @@ menu:
     weight: 5000
 section_menu: corda-5-beta
 ---
-This section describes the CLI commands and introduces RPC permission templates.  To learn how to insall the CLI, see [Installing the Corda CLI](../developing/getting-started/installing-corda-cli.md).
+This section describes the CLI commands and introduces REST permission templates. To learn how to insall the CLI, see [Installing the Corda CLI](../developing/getting-started/installing-corda-cli.md).
```

Comment:
not introduced by this change but since I'm reviewing it...
```suggestion
This section describes the CLI commands and introduces REST permission templates. To learn how to install the CLI, see [Installing the Corda CLI](../developing/getting-started/installing-corda-cli.md).
```